ddtrace/tracer/sampler.go

```diff
@@ -325,6 +329,15 @@ func (rs *rulesSampler) applyRate(span *span, rate float64, now time.Time) {
 	span.SetTag(keyRulesSamplerLimiterRate, rate)
 }
 
+// limit returns the rate limit set in the rules sampler, controlled by DD_TRACE_RATE_LIMIT, and
+// true if rules sampling is enabled. If not present it returns math.NaN() and false.
+func (rs *rulesSampler) limit() (float64, bool) {
+	if rs.enabled() {
+		return float64(rs.limiter.limiter.Limit()), true
```

Comment:
We should be OK without `nil` checks here. Other places throughout the rulesSampler code assume these fields are not `nil` and all of the constructors populate them. 

If we ever changed rulesSampler to allow `nil` members, we would be unlikely to miss this line, since it lives within the `rulesSampler` methods.